src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -55,6 +55,9 @@ public function getRules(): array
             'escape_implicit_backslashes' => true,
             'explicit_indirect_variable' => true,
             'explicit_string_variable' => true,
+            'fully_qualified_strict_types' => [
+                'import_symbols' => true,
+            ],
```

Comment:
FYI, changing rule configuration is not DX, it's feat.

changing `.php-cs-fixer.dist.php` config is DX/chore